app/assets/javascripts/sequence_show.js

```diff
@@ -105,9 +131,255 @@ function initSequenceShow(data) {
         }
     }
 
+    function setUpFA(fa) {
+        setUpGO(fa.go);
+        setUpEC(fa.ec);
+    }
+
+    function setUpEC({numAnnotatedPeptides, data}) {
+        let ecResultset = new ECNumbers({numAnnotatedPeptides, data});
```

Comment:
`let` can be `const` on various places in this file.

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13723) by @bmesuere on Mon Feb 19 2018 at 14:20._